docker/mlpack/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:latest
```

Comment:
I guess this is a matter of preference and scope, are we trying to replicate a dev setup or a minimal set up. In which case we could go for alpine linux or something similar, which would produce a smaller image size.

Reply:
I would agree with this.  The more minimal we can make the image, the easier it is to deploy in production.